menas/src/test/resources/application.properties

```diff
@@ -67,7 +67,8 @@ menas.environment=localhost
 
 menas.spark.master=local[1]
 
-spline.urlTemplate=http://localhost:8080/spline/dataset/lineage/_search?path=%s&application_id=%s
+menas.lineage.readApiUrl=//localhost:8080/spline-gw/consumer
```

Comment:
Is the leading `http` omitted intentionally (e.g. for supplying either `http` or `https`) or is this a mistake? If this is intentional, maybe there could be some comment explaining this?

Reply:
This is test properties. But in the main [properties](https://github.com/AbsaOSS/enceladus/pull/1175/files/2d10c5fbe907c46e69b27ad53d52614c4cd96440#diff-b9e082d316531342a516d81f8f47380cR75) there is a comment.

Reply:
I will still add it the comment here as well, when moving to 0.5. 👍